lxd/instance/instance_utils.go

```diff
@@ -302,10 +302,8 @@ func LoadInstanceDatabaseObject(ctx context.Context, tx *db.ClusterTx, project,
 	var container *cluster.Instance
 	var err error
 
-	if strings.Contains(name, shared.SnapshotDelimiter) {
-		parts := strings.SplitN(name, shared.SnapshotDelimiter, 2)
-		instanceName := parts[0]
-		snapshotName := parts[1]
+	if shared.IsSnapshot(name) {
```

Comment:
Maybe we should just use `api.GetParentAndSnapshotName` everywhere (or at least in more places) and update it to use strings.Cut. This way we don't have to repeat the check for `/` in the string.